src/features/saksoversikt/sak.slice.ts

```diff
@@ -180,6 +180,22 @@ export const lagreEpsGrunnlag = createAsyncThunk<
     return thunkApi.rejectWithValue(res.error);
 });
 
+export const lagreEpsGrunnlagSkjermet = createAsyncThunk<
+    unknown,
+    {
+        sakId: string;
+        behandlingId: string;
+        epsFnr: string;
+    },
+    { rejectValue: ApiError }
+>('behandling/grunnlag/bosituasjon/eps/skjermet', async (arg, thunkApi) => {
+    const res = await behandlingApi.lagreGrunnlagEpsSkjermet(arg);
+    if (res.status === 'ok') {
+        return res.data;
+    }
+    return thunkApi.rejectWithValue(res.error);
+});
+
```

Comment:
Kan vurdere om vi ens trenger å dispatche en action for dette? Räcker ikke det med att bara kjøre iväg ett api-kall? 😊

Reply:
Osäker, jeg vet egentlig ikke hva forskjellen er..?

Reply:
cc @ingara

Reply:
Hm, godt spørsmål egentlig. Sånn som det er nå, hvor denne ikke håndteres i reducer så kunne man nok tenkt at det like gjerne bare var et API-kall ja.
Samtidig så gjør vi jo noen ekstra greier call-site med å resette `sak` og `søker` fordi saksbehandler nå ikke skal ha tilgang til saken lenger. Kanskje det egentlig burde ha blitt gjort i de respektive reducerne/slicene i stedet? Hva tenker dere?

Reply:
Generelt tenker jeg at mindre kode er bedre enn mer kode, så hvis vi kan fjerne et av lagene her uten noen ulemper så høres jo det ganske bra ut.